docs/api-reference/retention-rules-api.md

```diff
@@ -25,45 +25,464 @@ sidebar_label: Retention rules
 
 This document describes the API endpoints for managing retention rules in Apache Druid.
 
-## Retention rules
+Druid uses retention rules to determine what data is retained in the cluster. Druid supports load, drop, and broadcast rules. See [using rules to drop and retain data](../operations/rule-configuration.md) for more information. 
 
-Note that all _interval_ URL parameters are ISO 8601 strings delimited by a `_` instead of a `/` as in `2016-06-27_2016-06-28`.
+In this document, `http://SERVICE_IP:SERVICE_PORT` is a placeholder for the server address of deployment and the service port. For example, on the quickstart configuration, replace `http://ROUTER_IP:ROUTER_PORT` with `http://localhost:8888`.
```

Comment:
I don't really understand the need for the double placeholder explanation when the router one is already a placeholder?

What does it do that the following doesn't?

`http://ROUTER_IP:ROUTER_PORT` is a place holder for your Router service address and port. Replace it with the information for your deployment. For example, use `http://localhost:8888` for quickstart deployments.